schemas/view-schema.json

```diff
@@ -194,38 +194,26 @@
                 }
               },
               "readOnly": true
-            }
-          },
-          "readOnly": true
-        },
-        "focus": {
-          "type": "object",
-          "description": "Focus properties",
-          "properties": {
-            "focused_times": {
+            },
+            "in_foreground_times": {
               "type": "array",
-              "description": "List of the time the user had the document focused for browser",
+              "description": "List of the period the user had the view in foreground (focused in the browser)",
```

Comment:
is this a browser only property? this can be translated into mobile apps too as they can go background<->foreground

Reply:
For the time being, it's only for browser but it should be usable as well from a mobile point of view.
At least, that's the idea.